pkg/scalers/prometheus_scaler.go

```diff
@@ -251,7 +239,7 @@ func (s *prometheusScaler) Close(context.Context) error {
 }
 
 func (s *prometheusScaler) GetMetricSpecForScaling(context.Context) []v2.MetricSpec {
-	metricName := kedautil.NormalizeString(fmt.Sprintf("prometheus-%s", s.metadata.metricName))
```

Comment:
@JorTurFer Do you have any context why we have another extra `prometheus` prefix here . I will just remove it. wdyt?